tests/functional/modules/test_zos_tso_command_func.py

```diff
@@ -146,7 +114,8 @@ def test_zos_tso_command_invalid_command(ansible_zos_module):
     hosts = ansible_zos_module
     results = hosts.all.zos_tso_command(commands=["xxxxxx"])
     for result in results.contacted.values():
-        assert result.get("output")[0].get("rc") == 255
+        for item in result.get("output"):
+            assert item.get("rc") == 255
         assert result.get("changed") is False
 
 
```

Comment:
I know you didn't touch this next test `test_zos_tso_command_multiple_commands` but I was wondering if it's possible to verify that 2 commands were run. Currently it looks like the test is only checking that the `rc` is `0`. I'd like to see a check for the correct number of `rc`s coming back (from [docs](https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/modules/zos_tso_command.html#return-values), it appears that for each command there should be a return structure).